src/components/select/select.tsx

```diff
@@ -255,11 +255,30 @@ export class Select {
     ) {
         event.stopPropagation();
 
+        // Construct the selector for the list element
+        const selector = `#${this.portalId} limel-menu-surface limel-list`;
+        const list: HTMLElement = document.querySelector(selector);
+
+        if (!list) {
+            return;
+        }
+
+        const scrollPosition = list.scrollTop;
+        const delay = 8;
+
+        // Verify if we can manually manipulate the scroll position
```

Comment:
Same here. If you need to clarify something to the reviewer, just add a comment here on GH.